### DISCUSSION TITLE
Is there any way to get "batch convert" to only import English subs so I don't have to remove the rest by hand?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5372] [General]
![image](https://user-images.githubusercontent.com/82276985/135904465-5e0cd0e8-1b6c-418f-a0e7-c658379a0881.png)

**Comment by OmrSi** [comment]
Hello, @DrReddP 

Yes, there is.
Use `Filter` with:
![image](https://user-images.githubusercontent.com/20923700/135905577-59db1a27-9e42-4051-9e5a-aebd31221a99.png)

If you're using the latest beta, you can use it even after the files are imported, if you're using an older version, you need to activate it then import the files.

**Comment by DrReddP** [comment]
Perfect, thanks
